impl/src/main/java/org/ehcache/internal/serialization/CompactPersistentJavaSerializer.java

```diff
@@ -51,13 +48,8 @@ public CompactPersistentJavaSerializer(ClassLoader classLoader, FileBasedPersist
     }
   }
 
-  @Override
-  public final void close() throws IOException {
```

Comment:
Same as other comment, the full logic should remain, just called from `SerializationProvider.releaseSerializer` now.